packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -107,24 +112,19 @@ import type {
   WebAuthnAuthenticationVerificationParams,
   ILitNodeClient,
   GetPkpSessionSigs,
+  CapacityCreditsReq,
+  CapacityCreditsRes,
+  JsExecutionRequestBody,
+  JsonSignSessionKeyRequestV1,
+  BlsResponseData,
+  SessionKeyCache,
 } from '@lit-protocol/types';
+import * as blsSdk from '@lit-protocol/bls-sdk';
 
-// TODO: move this to auth-helper for next patch
-interface CapacityCreditsReq {
-  dAppOwnerWallet: ethers.Wallet;
-  capacityTokenId?: string;
-  delegateeAddresses?: string[];
-  uses?: string;
-  domain?: string;
-  expiration?: string;
-  statement?: string;
-}
+const TEMP_CACHE_PERIOD = 30000; // 30 seconds
```

Comment:
What is this cache used for? Note that a sessionSig can be used only for one request. For each new request the sessionKey needs to resign and create a new sessionSig @glitch003

Reply:
This is fixed here: https://github.com/LIT-Protocol/js-sdk/pull/446